### DISCUSSION TITLE
AVG blocks Lib 6.1.1 zip download as URL is Black Listed

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#2844] [Q&A] [answered]
Was not able to download Lib 6.1.1 zip as AVG blocks it (see screenshot). Is this normal and should I turn off AVG to download? 
![image](https://github.com/dcs-liberation/dcs_liberation/assets/94265170/66233059-6d9a-4378-907f-dd06f07c2b3f)

**Comment by Starfire13** [comment, marked as answer]
It's a false positive. You can disable AVG and download it.
